docs/management/cases/setup-cases.asciidoc

```diff
@@ -0,0 +1,27 @@
+[[setup-cases]]
+== Configure access to cases
+
+preview::[]
+
+To access cases in *{stack-manage-app}*, you must have the appropriate {kib}
+privileges:
+
+[options="header"]
+|=== 
+
+| Action | {kib} privileges
+| Give full access to manage cases 
+a|
+* `All` for the *Cases* feature under *Management*.
+* `All` for the *Actions and Connectors* feature under *Management*.
+
+NOTE: Roles without `All` *Actions and Connectors* feature privileges cannot
```

Comment:
Maybe it would be nice to add that you cannot also send updates to external systems if you do not have the `All` feature privilege.